Formatting, table fixes

IanWhyte opened the pull request:
Includes fixes that enable tables to display properly.

mojavelinux commented on anypoint-b2b/v/latest/document-types.adoc:
```diff
@@ -447,14 +451,14 @@ None
 
 [%header,cols="3s,7a"]
+
 |===
```
Please don't include blank lines between the block attributes and the opening table delimiter.

```
[%header,cols="3s,7a"]
|===
```

instead of

```
[%header,cols="3s,7a"]

|===
```

Thanks!

IanWhyte commented:
Thanks for the input, Dan.

I tend to include a lot of blank lines in my code - it helps me see things better. Are there any other places you can think of where, in an .adoc file, that's an issue?

Ian

-----Original Message-----
From: "Dan Allen" <notifications@github.com>
Sent: ‎9/‎16/‎2016 7:05 PM
To: "mulesoft/mulesoft-docs" <mulesoft-docs@noreply.github.com>
Cc: "Ian Whyte" <ian@modusbox.com>; "Author" <author@noreply.github.com>
Subject: Re: [mulesoft/mulesoft-docs] Formatting, table fixes (#800)

@mojavelinux commented on this pull request.



In anypoint-b2b/v/latest/document-types.adoc:
> @@ -447,14 +451,14 @@ None
 == Format (JSON)
 
 [%header,cols="3s,7a"]
+
 |===
Please don't include blank lines between the block attributes and the opening table delimiter.
[%header,cols="3s,7a"]
|===
instead of
[%header,cols="3s,7a"]

|===
Thanks!
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

mojavelinux commented:
In many cases, blank lines have meaning, so you have to be careful where you use them. The general rule of thumb is as follows:

* no blank line between a block and its block attributes
* one blank line before and after a section title (where section title includes the block attributes)
* one blank line between paragraphs and delimited blocks
* blank line between table rows, but not between cells in the same row*
  - I also recommend putting each table cell on its own line, except for cells in the header row
* no blank lines within a list item that has complex content (i.e., a list item that uses list continuations)

\* Keep in mind that a blank line after the first row in a table makes that row a header row

If you have a question about a specific case, don't hesitate to ask (dan [at] opendevise [dot] io).

IanWhyte commented:
Thanks, Dan. I've fixed that occurrence of this issue. Is the next step yours?

IanWhyte commented:
Thanks, Dan, that's useful information.

About the mechanics of working with PR 800: I've fixed the issue and added
a comment indicating I've done so. Will you see the comment if you're not
actually looking for it? Meaning, what are the appropriate steps for
addressing comments in a PR on the way to accepting it?

Thanks, Ian

On Fri, Sep 16, 2016 at 8:06 PM, Dan Allen <notifications@github.com> wrote:

> In many cases, blank lines have meaning, so you have to be careful where
> you use them. The general rule of thumb is as follows:
>
>    - no blank line between a block and its block attributes
>    - one blank line before and after a section title (where section title
>    includes the block attributes)
>    - one blank line between paragraphs and delimited blocks
>    - blank line between table rows, but not between cells in the same
>    row*
>       - I also recommend putting each table cell on its own line, except
>       for cells in the header row
>    - no blank lines within a list item that has complex content (i.e., a
>    list item that uses list continuations)
>
> * Keep in mind that a blank line after the first row in a table makes that
> row a header row
>
> If you have a question about a specific case, don't hesitate to ask (dan
> [at] opendevise [dot] io).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mulesoft/mulesoft-docs/pull/800#issuecomment-247745683>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANjA91suRo4jCa65TbUz2yoyIT5O1tbGks5qq1kmgaJpZM4J-X7B>
> .
>

mojavelinux commented:
> I've fixed that occurrence of this issue. Is the next step yours?

As I'm a general adviser, you don't need sign off from me. I'm just offering suggestions / guidelines that I think are going to help keep the source tidy and consistent. As I can ask is that you incorporate those suggestions, as you have done. Thanks!

george-hoffer commented:
Thanks Ian
